### PR TITLE
Issue with production environment installing kubeadm

### PR DESCRIPTION
**Pull request by RA489**
Issue with production environment installing kubeadm
Refs https://github.com/kubernetes/website/issues/15854

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -252,6 +252,10 @@ If you are using a different CRI, you have to modify the file
 ```bash
 KUBELET_EXTRA_ARGS=--cgroup-driver=<value>
 ```
+{{< warning >}}
+Flag --cgroup-driver has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: If `--cgroup-driver` is a deprecated flag, then the documentation shouldn't recommend setting it (on line 253).

This doesn't look like the right fix.

Comment (neolit123) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: i'd say omit this change for now.

while some of the kubelet flags are deprecated there is no clear timeline when they are going to be removed. so, once the time comes we will update both their handling in kubeadm and the documentation.

Comment (RA489) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: sure

**Comment by xiangpengzhao** [comment]
@RA489 seems like we might want to close this PR for now as per the inline comments?

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: @RA489 any update on this?

**Comment by zacharysarah** [comment]
Closing per https://github.com/kubernetes/website/issues/15854#issuecomment-521296268.

/close

**Pull request closed without merging**